### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Krishnabot**
In this pull request:

1. HTML, CSS, and javascript Linters are added.
2. HTML, CSS, and javascript codes are tested.
3. Functionality added When the user clicks the hamburger button, the mobile menu appears.
4. Functionality added When the user clicks the close (X) button, the mobile menu disappears.
5. Functionality added When the user clicks any of the mobile menu options, the mobile menu disappears.
6. Functionality added When the user clicks any of the mobile menu options, a correct part of the page is displayed.
7. live Demo of the project added. 
.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +15,9 @@
...
              <a href="#" class="logo">My Logo</a>
             <ul class="nav-menu">
-                <li class="nav-item"><a class = "nav-link" href="#">Portfolio</a></li>
-                <li class="nav-item"><a class = "nav-link" href="#">About</a></li>
-                <li class="nav-item"><a class = "nav-link" href="#">Contact</a></li>
+                <li class="nav-item"><a class = "nav-link portfolio-link" href="#portfolio">Portfolio</a></li>
+                <li class="nav-item"><a onclick="myFunction()" class =  " nav-link about-link" href="#about">About</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#contact">Contact</a></li>
             </ul>
             <div class="hamburger">
```
Comment (OLIPLICHE) [inline index.html:21]: - [ ] Please your hamburger button and menu should follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins).
- Your Design
 b

- Original Design   
c

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Krishnabot , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] You can also consider:

- Please adjust the margins, and paddings, and make the `header` look like the original one.
- Your image: 
a
- Original 
d



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      navLink.classList.toggle('active');
+    });
+  }
+}
+if (hamburger) {
+  hamburger.addEventListener('click', () => {
+    hamburger.classList.toggle('active');
+    navMenu.classList.toggle('active');
+    navBar.classList.toggle('active');
+    myFunction();
```
Comment (sja-thedude) [inline script.js:19]: - [ ] Kindly make sure to follow the background for the menu as per Figma design. You should use backdrop to have the same effect.
- [ ] Kindly make sure that for menu and for hamburger you follow the margin as given in the design. 
- [ ] Make sure to have margin to the left,right and top for the portfolio.
- [ ] Make sure to add top and right margin for mobile menu.

https://user-images.githubusercontent.com/70269507/174003018-26ce006b-5a68-4e3d-99bc-5a3a9a4dd220.mov

https://user-images.githubusercontent.com/70269507/174002973-89914ff3-852a-49e1-9bf9-5e12f738c889.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Krishnabot, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good jo so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Krishnabot,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] kindly fix the about section in your next milestone use **padding-left**

Capturepasd

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**